.gitignore

```diff
@@ -52,4 +52,6 @@ pkg/
 *.tmp.js
 
 # Used to see output of babel.
-/lab
\ No newline at end of file
+/lab
+
+*.mm_profdata
```

Comment:
unsure if it's intended but `npm run build:dev` always run `-- -Zself-profile`. Either case I don't expect we'll check in this profile ever.